migration/schema/v1/tables.go

```diff
@@ -1011,3 +1011,39 @@ func ConvertV2SamlToIdpConfig(samlMap map[string]any) (v3Config *IdpConfig, err
 	v3Config.Enabled = true
 	return
 }
+
+// Group is respretation of a row in "platformdb"."groups" table
+type Group struct {
+	GroupID     string `json:"group_id,omitempty"`
+	DisplayName string `json:"display_name,omitempty"`
+	RealmID     string `json:"realm_id,omitempty"`
+}
+
+func ConvertToGroup(grp map[string]any) (g *Group) {
+	group := &Group{}
+	group.GroupID = grp["_id"].(string)
+	group.DisplayName = grp["displayName"].(string)
+	group.RealmID = "defaultSP"
+	return group
+}
+
+// UserGroup is respretation of a row in "platformdb"."users_groups" table
+type UserGroup struct {
```

Comment:
I assume we're still waiting on an insert UserGroup implementation, correct?

Reply:
That's correct.